Codec/Picture/Extra.hs

```diff
@@ -130,3 +133,38 @@ rotateRight90 img@Image {..} =
   where
     gen x y = pixelAt img y (imageHeight - 1 - x)
 {-# INLINEABLE rotateRight90 #-}
+
+
+-- | Rotate image by 180, i.e flip both vertically and horizontally.
+
+rotate180 :: Pixel a => Image a -> Image a
```

Comment:
Sorry, my fault, for some reason I thought they should differ, but now I see that they in fact do not. Please ignore this comment.